src/scr.c

```diff
@@ -845,30 +956,65 @@ static int scr_get_params()
     tmp = kvtree_set_kv(scr_storedesc_hash, SCR_CONFIG_KEY_STOREDESC, scr_cntl_base);
     kvtree_util_set_int(tmp, SCR_CONFIG_KEY_COUNT, 0);
   }
+  else { // 
+    if (scr_my_rank_world == 0) {
+      // 23. Dump SCR parameters - SCR_CONFIG_KEY_STOREDES 
+      // kvtree_print(scr_storedesc_hash, 4); 
+      kvtree_print_mode(scr_storedesc_hash, 4, KVTREE_PRINT_KEYVAL);
+    }
+  }
 
   /* select copy method */
   if ((value = scr_param_get("SCR_COPY_TYPE")) != NULL) {
     if (strcasecmp(value, "single") == 0) {
       scr_copy_type = SCR_COPY_SINGLE;
+      if (scr_my_rank_world == 0) {
+         // 24. Dump SCR parameters - SCR_COPY_SINGLE
+         scr_dbg(scr_debug, "copy method, SCR_COPY_SINGLE");
+      }
```

Comment:
When you move this print outside of the outer ``if`` block, you can drop the other blocks that currently follow in the various ``else`` blocks.  So that should reduce down to just a single scr_dbg block after the bottom of the outer ``if`` block.